webui/react/src/contexts/Store.tsx

```diff
@@ -139,7 +138,7 @@ const initClusterOverview: ClusterOverview = {
   [ResourceType.UNSPECIFIED]: clone(initResourceTally),
 };
 const initInfo = {
-  branding: BrandingType.Determined,
+  branding: '' as BrandingType,
```

Comment:
**suggestion:** this makes me a little bit nervous because it's forcing TypeScript to acknowledge an empty string as a branding type. If there are areas in the code using this value and expecting HPE/Determined values but uses the empty string, things can fail poorly if uncaught.

My recommendation is to leave it as `BrandingType.Determined` and checking the `checked` field (may require some tweaking to ensure it is set to `true` properly). Then in areas referencing the branding type, before rendering, checking against that bit to make sure it is true before rendering the appropriate UI.

Reply:
Another alternative approach would be to use `branding?: BrandingType;` (allow it to be `undefined` and checking that the value is set). The downside is that this might require a lot of changes across the codebase (basically anywhere branding is used)